mix.exs

```diff
@@ -22,7 +22,7 @@ defmodule KittenBlue.Mixfile do
   # Run "mix help compile.app" to learn about applications.
   def application do
     [
-      extra_applications: [:logger, :jose, :poison]
+      extra_applications: [:logger]
```

Comment:
We don't need to specify every Elixir dependencies here.
cf. https://www.amberbit.com/blog/2017/9/22/elixir-applications-vs-extra_applications-guide/